third-party/gasnet/Makefile.setup

```diff
@@ -32,7 +32,7 @@ endif
 
 # Remove -Winline from Gasnet's provided flags
 CHPL_GASNET_CFLAGS = $(CHPL_GASNET_CFLAGS_ALL:-Winline=)
-ifneq (, $(filter $(CHPL_MAKE_TARGET_PLATFORM),cray-xt cray-xe cray-xk cray-xc) )
+ifneq (, $(filter $(CHPL_MAKE_TARGET_PLATFORM),cray-xt cray-xe cray-xk cray-xc cross-aarch64) )
```

Comment:
Does aarch64 really need -DGASNET_NEEDS_MAX_SEGSIZE?  I think that again this should not be combined with cray-x-series stuff.